src/gui/screen_menu_esp_update.cpp

```diff
@@ -153,6 +179,15 @@ void ScreenMenuESPUpdate::windowEvent(EventLock /*has private ctor*/, window_t *
                 f_close(&file_descriptor);
             }
             progress_state = esp_upload_action::Connect;
+            /*
+             * Don't leave on timeout when flashing, which would cancel the
+             * flashing (but be willing to time out before that if the user
+             * just wonders into the menu.
+             *
+             * This flag is not reset. We want the user to be able to see the
+             * result.
+             */
+            flags.timeout_close = is_closed_on_timeout_t::no;
```

Comment:
move this to ctor

Reply:
This was here kind of on purpose ‒ the intention was that if the user doesn't start flashing, the menu times out the usual way.

Should I still move it?